sourcetracker/_cli/gibbs.py

```diff
@@ -19,11 +19,12 @@
 from ipyparallel import Client
 
 from sourcetracker._cli import cli
-from sourcetracker._sourcetracker import (biom_to_df, _gibbs, _gibbs_loo,
-                                          intersect_and_sort_samples,
+from sourcetracker._sourcetracker import (gibbs, intersect_and_sort_samples,
                                           get_samples, collapse_source_data,
-                                          subsample_dataframe)
-from sourcetracker._util import parse_sample_metadata
+                                          subsample_dataframe,
+                                          validate_gibbs_input, plot_mpm)
+
+from sourcetracker._util import parse_sample_metadata, biom_to_df
 
 
 @cli.command(name='gibbs')
```

Comment:
I like leaving the `--loo` flag because: the user is passing only a single table. We are doing the splitting for them. If they wanted to do normal source/sink prediction, then switch to LOO on the same samples, they'd have to modify their mapping file without the LOO flag. I think it will be unfortunately easy for users to screw up the passing the right combination of `source_sink_column` etc. to get the functionality of the easy `--loo`.

If you feel strongly about this one @gregcaporaso let's chat and we can come to something.